### DISCUSSION TITLE
Permissions on dashboard variables

### DISCUSSION THREAD

**Discussion by techgnosis** [grafana/grafana#41163] [Dashboards, Panels & Visualizations]
This request comes from a current Grafana Enterprise customer.

The customer wants to be able to assign permissions to dashboard variables. This would allow the customer to share dashboards among their users and then limit each Grafana user to select the variable for their data. Right now you cannot permission dashboard variables, so if you want your users to only be able to select their specific dashboard variable, you have to create a dashboard per variable and permission the dashboards themselves.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
